packages/dev/s2-docs/pages/internationalized/date/Calendar.mdx

```diff
@@ -15,6 +15,7 @@ import {StaticTable} from '../../../src/StaticTable';
 
 export const section = 'Date and Time';
 export const group = 'Internationalized';
+export const description = 'Calendar systems for international date calculations.';
```

Comment:
<img width="993" height="270" alt="Image" src="https://github.com/user-attachments/assets/3dbe7f08-db6d-4343-9b79-bb7100a43fe1" />

Now that we have descriptions for each component, could we have the descriptions start from right beneath the component name?